spec/models/alaveteli_pro/user_phase_counts_spec.rb

```diff
@@ -71,6 +71,25 @@
 
     end
 
+    it 'caches the results' do
+      before = user.phase_count('awaiting_response')
+      AlaveteliPro::RequestSummary.
+        create_or_update_from(FactoryGirl.create(:info_request, user: user))
+      expect(user.phase_count('awaiting_response')).to eq(before)
```

Comment:
This isn't a great test, because `not_drafts` is currently not memoized and would therefore fail.

There's a design problem with this this module which I've addressed in the main review section, but this test should effectively be:

```ruby
expect(user.phase_counts).to eq({ awaiting_response: 2, not_drafts: 3 })
```